apps/meteor/server/database/DatabaseWatcher.ts

```diff
@@ -218,15 +223,16 @@ export class DatabaseWatcher extends EventEmitter {
 
 		this.lastDocTS = new Date();
 
-		this.metrics?.oplog.inc({
-			collection,
-			op: doc.action,
-		});
+		this.emit('doc', { collection, doc });
 
 		this.emit(collection, doc);
 	}
 
-	on<T>(collection: string, callback: (event: RealTimeData<T>) => void): this {
+	onDoc(callback: (event: { collection: string; doc: RealTimeData<IRocketChatRecord> }) => void): this {
```

Comment:
not a fan of `onDoc`.. it could also be `onDocument` or maybe it shouldn't exist? it exists basically to have typings on the subscribing.

Reply:
Vote for `onDocument`, or also `onOplogEntry`

Reply:
renamed to `onDocument`